test/controllers/projects/transfer_controller_test.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+module Projects
+  class TransferControllerTest < ActionDispatch::IntegrationTest
+    setup do
+      sign_in users(:john_doe)
+      @project = projects(:project1)
+      @namespace = namespaces_user_namespaces(:john_doe_namespace)
+      @old_namespace = groups(:group_one)
+    end
+
+    test 'should transfer project' do
+      post namespace_project_transfer_index_path(@old_namespace, @project),
+           params: { new_namespace_id: @namespace.id }, as: :turbo_stream
+
+      assert_response :redirect
+    end
+
+    test 'should not create a new transfer with wrong parameters' do
+      post namespace_project_transfer_index_path(@old_namespace, @project),
+           params: { new_namespace_id: 0 }, as: :turbo_stream
+
+      assert_response :unprocessable_entity
+    end
+
+    test 'should not transfer a project to unowned namespace' do
+      post namespace_project_transfer_index_path(@old_namespace, @project),
```

Comment:
Shouldn't this also use `as: :turbo_stream`

Reply:
Makes sense. Updated in 6179663